src/schema/paddle.ts

```diff
@@ -148,13 +152,37 @@ export const resolvers: IResolvers<unknown, AuthContext> = traceResolvers<
       const featureValue: Record<string, string> =
         growthbookClient.getFeatureValue('pricing_ids', {});
 
-      const pricePreview = await paddleInstance?.pricingPreview.preview({
-        items: Object.keys(featureValue).map((priceId) => ({
-          priceId,
-          quantity: 1,
-        })),
-        address: region ? { countryCode: region as CountryCode } : undefined,
-      });
+      const hmac = createHmac('sha1', process.env.SLACK_SIGNING_SECRET);
+      hmac.update(Object.keys(featureValue).toString());
+      const pricesHash = hmac.digest().toString('hex');
```

Comment:
We hash the prices so we don't have to cache paddle cores per user id and all pricing ids